jms/1.7/modules/ROOT/pages/index.adoc

```diff
@@ -1,287 +1,70 @@
 = JMS Connector - Mule 4
-:page-aliases: connectors::jms/jms-connector.adoc
+:page-aliases: connectors::jms/jms-connector.adoc, connectors::jms/jms-about.adoc
 
 Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
 
-JMS Connector v1.7
+Anypoint Connector for JMS (Java Message Service) (JMS Connector) enables sending and receiving messages to queues and topics for any message service that implements the JMS specification. JMS is a widely-used API for message oriented middleware. It enables the communication between different components of a distributed application to be loosely coupled, reliable, and asynchronous. The JMS Connector main features include:
 
-Anypoint Connector for JMS (JMS Connector) enables your app to do messaging using the JMS implementation of your choice. Its main features include:
-
-* Pub/Sub pattern support on any given destination.
-* Listen/Reply pattern support on any given destination.
-* Publish-Consume pattern support on any given destination, with fixed or temporary reply Queue.
+* Publish and subscribe pattern support on any given destination.
+* Listen and reply pattern support on any given destination.
+* Publish-Consume pattern support on any given destination, with a fixed or temporary reply queue.
```

Comment:
I don't know about this Publish-Consume. For parallel structure, should it be like the others and just say Publish and consume pattern? OR make the others like this one Publish-Subscribe, Listen-Reply...
Although, I also don't know about the caps. Should it be Publish-consume or Publish-Consume? idk